templates/zerver/integrations.html

```diff
@@ -2158,7 +2166,93 @@
     </div>
 
     <div id="trello" class="integration-instructions">
+      <p>This integration is for the hosted version of Trello.
+        To integrate with a locally installed instance of Trello, use the
+        <a href="#trello-plugin">plugin</a>.</p>
+
+      <p>
+        First, create the stream you'd like to use for Trello notifications,
+        and subscribe all interested parties to this stream. We recommend the
+        name <code>trello</code>.
+      </p>
+
+      <p>
+        Your webhook URL is:<br/>
+        <code>https://api.zulip.com/v1/external/trello?api_key=abcdefgh&stream=trello</code><br/>
+        where <code>api_key</code> is the API key of your Zulip bot,
+          and <code>stream</code> is the stream name you want the notifications sent to.
+      </p>
+
+      <p>
+        Before you create a webhook,
+        you'll need to follow steps below to get an <b>APPLICATION_KEY</b>, and a <b>UserToken</b>,
```

Comment:
@TomaszKolek 
`you'll need to follow steps below`

should be

`you'll need to follow the steps below`